vcbuild.bat

```diff
@@ -185,20 +181,19 @@ if %target_arch%==x64 if %msvs_host_arch%==amd64 set vcvarsall_arg=amd64
 
 @rem Look for Visual Studio 2017
 :vs-set-2017
-if defined target_env if "%target_env%" NEQ "vs2017" goto vs-set-2015
```

Comment:
This line should not be removed, it'll be needed again when we add a new VS version.

Reply:
It would need to be changed now anyway. And when we add a new VS version, it will need to be changed again. I'd rather remove it now and add it back if and when it's needed than keep a useless line around for likely at least a year.

That said, I don't really mind either way. If anyone else feels strongly about it, I'll put it back.

FWIW it was also removed in https://github.com/nodejs/node/pull/8067.

Reply:
A new version would go on top, so it would stay the same if changed now. I'd rather these blocks behaved more atomically, and I see this as part of the `vs2017` parameter. But it works either way, feel free to go ahead.